Changelog.md

```diff
@@ -1,481 +0,0 @@
-# Change Log
```

Comment:
Did you mean to delete this file?

Reply:
Oh, you moved it in to the README. That wasn't mentioned in the PR description and is a bit surprising but I assume you have done so deliberately.